app/views/_includes/applications/personal-details.njk

```diff
@@ -5,7 +5,7 @@
   {%- endfor -%}
 {% endset %}
 
-<h2 class="govuk-heading-m govuk-!-margin-top-6 govuk-!-margin-bottom-2 govuk-!-font-size-27">Personal details</h2>
+<h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-top-7 govuk-!-margin-bottom-2 ">Personal details</h2>
```

Comment:
We could make future iterations easier if we specify a heading class rather than an amalgamation of overrides